Header header
float32[] W
float32[] WN
---
bool success
string message